packages/firebase/rollup.config.js

```diff
@@ -70,7 +70,8 @@ const plugins = [
   commonjs()
 ];
 
-const external = Object.keys(pkg.dependencies || {});
+const deps = [...Object.keys(pkg.dependencies || {}), 'tslib'];
```

Comment:
You don't need to add `tslib` which is a not a direct dependency of `firebase/firestore/memory`, etc. We are good as long as the scoped packages e.g. `@firebase/firestore/memory` are excluded.

Reply:
Fixed.